examples/accounts/getAccount.tsx

```diff
@@ -100,7 +70,7 @@ export const getAccountItem = ({
     if (type === 'company' && parentId) {
       const parentAccount = accounts.find((a) => a.id === parentId);
 
-      return 'Org. nr: ' + uniqueId + ', del av ' + parentAccount?.name;
+      return '↳ Org. nr: ' + uniqueId + ', del av ' + parentAccount?.name;
```

Comment:
I think `↳` and other visual details that serve to align the design, should be part of the component itself.